projects/packages/my-jetpack/_inc/components/my-jetpack-screen/index.jsx

```diff
@@ -77,6 +77,11 @@ const GlobalNotice = ( { message, title, options } ) => {
  * @return {object} The MyJetpackScreen component.
  */
 export default function MyJetpackScreen() {
+	const [ welcomeFlowExperiment, setWelcomeFlowExperiment ] = useState( {
+		isLoading: false,
+		//variation: 'control', // Temporarily set variation to 'treatment' for testing.
+		variation: 'treatment',
+	} );
```

Comment:
I know this is for testing so it's not a big deal right now, but if possible I think we should handle the experiment entirely from the `WelcomeFlow` component if at all possible. I don't think the passing of props is necessary here. I haven't run an experiment yet so I can't be positive, but it should also be possible to get that value from the `EvaluationRecommendations` component as well I think 🤔